docs/configuring/README.md

```diff
@@ -1,474 +0,0 @@
-# Configuring ESLint
```

Comment:
I think this file needs to stay because the configuration options are based on the version of the code. Whenever someone adds a new configuration option, this needs to be changed.